packages/nocodb/src/db/field-handler/index.ts

```
@@ -362,4 +398,102 @@ export class FieldHandler implements IFieldHandler {
+              await handler.parseDbValue({
+                value: row[column.title],
+                row,
+                baseModel: baseModel,
+                column,
+                options: params.options,
+              })
+            ).value;
+          }
+        }
+      } else {
+        continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)